archive_viewer/table_models/axis_model.py

```diff
@@ -1,10 +1,74 @@
+from typing import Any
+from qtpy.QtCore import (Qt, QVariant, QPersistentModelIndex, QModelIndex)
+from pydm.widgets.baseplot import BasePlotAxisItem
 from pydm.widgets.axis_table_model import BasePlotAxesModel
 
 
 class ArchiverAxisModel(BasePlotAxesModel):
     """The data model for the axes tab in the properties section. Acts
     as a go-between for the axes in a plot, and QTableView items.
+
+    Parameters
+    ----------
+    plot : BasePlot
+        The model's associated plot widget
+    parent : QObject, optional
+        The model's parent, by default None
     """
+    def __init__(self, plot, parent=None) -> None:
```

Comment:
could you add the type hint for plot? thanks

Reply:
Resolved